src/main/kotlin/no/skatteetaten/aurora/boober/service/AuroraConfigService.kt

```diff
@@ -124,10 +130,14 @@ class AuroraConfigService(@TargetDomain(AURORA_CONFIG) val gitService: GitServic
         createValidatedAuroraDeploymentSpecs(AuroraConfigWithOverrides(auroraConfig), affectedAid)
         watch.stop()
 
-        val mapper = jacksonObjectMapper()
         val checkoutDir = getAuroraConfigFolder(auroraConfig.affiliation)
         watch.start("write file")
-        val prettyContent = mapper.writerWithDefaultPrettyPrinter().writeValueAsString(newFile.contents)
+
```

Comment:
Duplicated code (see save(AuroraConfig))

Reply:
Yupp. But this is not code I have changed in this PR. Can we create a separate task for fixing that duplication?

Reply:
Your call, really. But since you have touched this duplicated code in your implementation already and actually introduced more duplication (made the problem worse), I think it is fair to refactor this now. Risk has already been introduced and the code needs testing in both places in either case.